services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/switches/SwitchValidationSpec.groovy

```diff
@@ -361,43 +388,60 @@ class SwitchValidationSpec extends HealthCheckSpecification {
         }
     }
 
-    def "Able to get empty switch validate information from the intermediate switch(flow contains > 2 switches)"() {
+    def "Able to validate and sync a switch with missing transit rule"() {
         given: "Two active not neighboring switches"
         def switchPair = topologyHelper.getAllNotNeighboringSwitchPairs().find { pair ->
             def possibleDefaultPaths = pair.paths.findAll { it.size() == pair.paths.min { it.size() }.size() }
             //ensure the path won't have only OF_12 intermediate switches
-            !possibleDefaultPaths.find { path ->
+            def hasOf13Path = !possibleDefaultPaths.find { path ->
                 path[1..-2].every { it.switchId.description.contains("OF_12") }
             }
+            hasOf13Path && pair.src.ofVersion != "OF_12" && pair.dst.ofVersion != "OF_12"
         }
-        when: "Create an intermediate-switch flow"
+
+        and: "Create an intermediate-switch flow"
         def flow = flowHelper.randomFlow(switchPair)
         flowHelper.addFlow(flow)
-        def flowPath = PathHelper.convert(northbound.getFlowPath(flow.id))
 
-        then: "The intermediate switch does not contain any information about meter"
-        def switchToValidate = flowPath[1..-2].find { !it.switchId.description.contains("OF_12") }
-        def intermediateSwitchValidateInfo = northbound.validateSwitch(switchToValidate.switchId)
-        switchHelper.verifyMeterSectionsAreEmpty(intermediateSwitchValidateInfo)
+        when: "Delete created rules on the transit"
+        def involvedSwitches = pathHelper.getInvolvedSwitches(flow.id)
+        def transitSw = involvedSwitches[1]
+        northbound.deleteSwitchRules(transitSw.dpId, DeleteRulesAction.IGNORE_DEFAULTS)
 
-        and: "Rules are stored in the 'proper' section on the transit switch"
-        intermediateSwitchValidateInfo.rules.proper.findAll { !Cookie.isDefaultRule(it) }.size() == 2
-        switchHelper.verifyRuleSectionsAreEmpty(intermediateSwitchValidateInfo, ["missing", "excess"])
+        then: "Rule info is moved into the 'missing' section"
+        verifyAll(northbound.validateSwitch(transitSw.dpId)) {
+            it.rules.missing.size() == 2
+            it.rules.proper.findAll {
+                !Cookie.isDefaultRule(it)
+            }.empty //ingress rule is left untouched
```

Comment:
what does ` }.empty //ingress rule is left untouched` mean?
you removed this rule in 409L

Reply:
fixed. Was left during some copy paste